src/components/Table/Table.types.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2023 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { AnyObject as GenericRecord } from 'antd/es/_util/type'
+export type { TablePaginationConfig as Pagination } from 'antd'
```

Comment:
I would put a blank line between import and export, I thought they were all import 😅

Reply:
will do 👍🏼

Reply:
fixed ✅